luke/index.js

```diff
@@ -0,0 +1,23 @@
+// Q1.In one night, a movie theater sells tickets for 6450 dollars.Each ticket costs 15 dollars.How many tickets were sold ?
+let tiketsSold = 6450 / 15;
+console.log(tiketsSold);
+
+// Q2.Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let yearIncome = 500 * 12;
+console.log("$" + yearIncome);
+
+// Q3.Calculate the percentage of 17 / 30. Expected output: number %
+let percentage = (17 / 30) * 100;
```

Comment:
Good Job. Also there is another alternative to solve this question: 
let percentage = (17*100) / 30

Reply:
cool ill put both in :)